README.md

```diff
@@ -1,40 +1,55 @@
 <!-- [![Lint Charts](https://github.com/hdsingh-uptycs/kspm-helm-charts/actions/workflows/lint-charts.yml/badge.svg?branch=main)](https://github.com/hdsingh-uptycs/kspm-helm-charts/actions/workflows/lint-charts.yml) -->
```

Comment:
This is pointing to another repo, can you remove this line or add lint charts into the github wokflow?

Reply:
Removed this for now because github workflows will be added after charts have been merged.